packages/sdk/src/oracle/optimistic/types/state.ts

```diff
@@ -0,0 +1,157 @@
+import { JsonRpcSigner, BigNumber, Web3Provider, FallbackProvider } from "oracle/common/types/ethers";
```

Comment:
this file is almost identical to the previous state file specific to the current optimistic oracle contract.  some of this needs to be refactored to remove duplicate information shared between the clients.